Tests/KituraTests/TestCodableRouter.swift

```diff
@@ -179,11 +179,11 @@ class TestCodableRouter: KituraTest {
             
             .request("post", path: "/urlencoded", urlEncodedString: "encoding=valid&failed=match")
             .hasStatus(.unprocessableEntity)
-            .hasNoData()
+            .hasData()
```

Comment:
We ought to be checking the `Content-Type` of the response is `text/plain`

Reply:
Actually, there's precedent for us sending plain text data without setting the Content-Type.  This is fine for now.